config.js

```diff
@@ -17,5 +17,6 @@ export default async function getConfig(command, flags) {
         autoPublish: flags.autoPublish || !command,
         trustedTesters: flags.trustedTesters,
         deployPercentage: flags.deployPercentage,
+        maxAwaitInProgress: flags.maxAwaitInProgress || process.env.MAX_AWAIT_IN_PROGRESS || undefined,
```

Comment:
```suggestion
        maxAwaitInProgress: flags.maxAwaitInProgress
```

ENVs are only used for secrets, no need for fallbacks